coops-ui/src/views/AnnualReport.vue

```diff
@@ -452,12 +453,14 @@ export default {
             }
 
             // load Change of Address fields
-            const changeOfAddress = filing.changeOfAddress
-            if (changeOfAddress) {
-              if (changeOfAddress.deliveryAddress && changeOfAddress.mailingAddress) {
+            if (filing.changeOfAddress) {
+              const offices = filing.changeOfAddress.offices
+              if (offices.registeredOffice) {
```

Comment:
You need a check that "offices" isn't null before you dereference it.
`if (offices && offices.registeredOffice) {`

Reply:
100%, i missed the point of that one, my bad. Will correct.

Reply:
Changed!

Reply:
This fix needs to be made in AnnualReport.vue on line 458 also.

Reply:
Oops, you don't need both lines 456 and 458 😛

Reply:
Fixed.